libs/community/langchain_community/document_loaders/glue_catalog.py

```diff
@@ -0,0 +1,103 @@
+from typing import Any, Dict, Iterator, List, Optional
+
+from langchain_core.documents import Document
+
+from langchain_community.document_loaders.base import BaseLoader
+
+
+class GlueCatalogLoader(BaseLoader):
+    """Load table schemas from AWS Glue.
+
+    This loader fetches the schema of each table within a specified AWS Glue database.
+    The schema details include column names and their data types, similar to pandas dtype representation.
+
+    AWS credentials are automatically loaded using boto3, following the standard AWS method:
+    https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html
+
+    If a specific AWS profile is required, it can be specified and will be used to establish the session.
+    """
+
+    def __init__(
```

Comment:
Consider allowing a user to pass in a boto3 session instance directly

Reply:
Nice idea. I just did it.